### PR TITLE
CLM-22306 add empty job 

### PR DESCRIPTION
**Pull request by isolonyi-sonatype**
Jira: [CLM-22306](https://issues.sonatype.org/browse/CLM-22306)
Build:

### CODE REVIEW

File: plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy [hunk 1]
```diff
@@ -113,6 +113,7 @@ download {
   }
 }
 
+
```
Comment (DanielRuaMadrid) [inline plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy:116]: Revert added empty line

File: plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy [hunk 2]
```diff
@@ -124,9 +125,9 @@ storage {
     this.configReloadLock.readLock().lock()
     try {
       runHandlerWithInitialisation(
-        runOnceBeforeAllHandlers: { asSystem { this.nexusFirewallForArtifactory.verifyInit() }},
-        handler: { nexusFirewallForArtifactory.afterDeleteHandler(item.repoPath) },
-        onRunOnceBeforeAllHandlersFailure: { this.log.error('afterDeleteHandler - ' + VERIFY_INIT_TIMEOUT) }
+          runOnceBeforeAllHandlers: { asSystem { this.nexusFirewallForArtifactory.verifyInit() } },
```
Comment (DanielRuaMadrid) [inline plugin/src/main/java/com/sonatype/iq/artifactory/nexusFirewallForArtifactoryPlugin.groovy:128]: Revert this changes. It looks like you IDE added empty white spaces

**Comment by vladt** [comment]
Please add a link to a successful Jenkins build for the branch for this PR.
